staging/src/k8s.io/kubectl/pkg/cmd/label/label_test.go

```diff
@@ -173,7 +173,20 @@ func TestLabelFunc(t *testing.T) {
 					Labels: map[string]string{"a": "b"},
 				},
 			},
-			labels:    map[string]string{"a": "b"},
+			labels: map[string]string{"a": "b"},
+			expected: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"a": "b"},
+				},
+			},
+		},
+		{
+			obj: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"a": "b"},
+				},
+			},
+			labels:    map[string]string{"a": "c"},
 			expectErr: true,
```

Comment:
Can you improve the test file so that it take expectErr as a `string` instead of a `boolean` value. That would give us a clear sense of what error we are getting

https://github.com/kubernetes/kubernetes/pull/101646#discussion_r640150201

Reply:
Done!